keras_cv/models/vgg19.py

```diff
@@ -14,50 +14,35 @@
 # ==============================================================================
 
 """VGG19 model for KerasCV.
-
 Reference:
   - [Very Deep Convolutional Networks for Large-Scale Image Recognition](https://arxiv.org/abs/1409.1556) (ICLR 2015)
 """
-
 import tensorflow as tf
 from tensorflow import keras
 from tensorflow.keras import layers
 
 from keras_cv.models import utils
+from keras_cv.models.vgg16 import apply_vgg_block
 
 
-def VGG19(
-    *,
-    include_rescaling,
-    include_top,
-    classes=None,
-    weights=None,
-    input_shape=(224, 224, 3),
-    input_tensor=None,
-    pooling=None,
-    classifier_activation="softmax",
-    name="VGG19",
-    **kwargs,
-):
-    """Instantiates the VGG19 architecture.
-
+@keras.utils.register_keras_serializable(package="keras_cv.models")
+class VGG19(keras.Model):
+    """
     Reference:
     - [Very Deep Convolutional Networks for Large-Scale Image Recognition](https://arxiv.org/abs/1409.1556) (ICLR 2015)
-
-    This function returns a Keras VGG19 model.
-
+    This class returns a Keras VGG19 model.
```

Comment:
How about "represents" instead of "returns"

Reply:
Fixed.